.github/resources/helm_it_role_binding.yaml

```diff
@@ -0,0 +1,14 @@
+# Permissions required by the GitHub Actions Runner to perform integration tests
```

Comment:
I'm looking for feedback on the ClusterRole and ClusterRoleBinding yaml files.  Should these be merged in?  They are run against the self hosted EKS cluster, but only once.  If we wanted to create a new EKS cluster to host our build jobs then these would come in handy.  Just not sure where to put them.

Reply:
I'm OK with the permissions as is as long as they're bound to the GitHub service account namespace, which seems to be the case.

As for where to put these - I've seen conventions on other teams where repos are stood up for architecture design files like yaml and terraform stuff. I'm OK where they are now though as it gives new users an easy basis for setting up their own EKS clusters for testing against forks -- maybe just provide documentation in the README about what they should be used for.

Reply:
I kind of like the idea of storing this in a IAC repo.  I'm going to look into it.

Reply:
I decided to yank them from this PR.  FYI